### ISSUE TITLE
Custom font for Dialog

### ISSUE THREAD

**Issue by iRoachie** [aakashns/react-native-dialogs#127]
In the original library, changing of the font family is supported by editing styles.xml https://github.com/afollestad/material-dialogs/blob/master/documentation/CORE.md#fonts

```xml
<style name="AppTheme.Custom" parent="Theme.AppCompat">
  <item name="md_font_title">@font/your_font</item>
  <item name="md_font_body">@font/your_font</item>
  <item name="md_font_button">@font/your_font</item>
</style>
```

However, this doesn't seem to work with this lib (android studio can't find the resource to link to).

How can font family be changed at the moment?

**Comment by Noitidart** [comment]
I think the original design plan for not including custom fonts is that we wanted to match iOS style dialogs in that these dialogs are always that system font.

**Comment by iRoachie** [comment]
Yes but then the dialogs look out of place since everything else in the app has a custom font. Custom fonts are even allowed in the original library.
